### PR TITLE
Add Degenpad

### PR DESCRIPTION
**Pull request by ThePanduuh**
## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/19812

## Checklist

- [x] The VIA support for this keyboard is in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/thepanduuh/degenpad.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Degenpad",
+  "vendorId": "0x0000",
+  "productId": "0x0000",
```
Comment (MaiTheSan) [inline v3/thepanduuh/degenpad.json:4]:
```suggestion
  "productId": "0x4447",
```

File: v3/thepanduuh/degenpad.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "Degenpad",
+  "vendorId": "0x0000",
```
Comment (MaiTheSan) [inline v3/thepanduuh/degenpad.json:3]:
```suggestion
  "vendorId": "0x4A44",
```

File: src/thepanduuh/degenpad/degenpad.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
+{
+  "name": "Degenpad",
+  "vendorId": "0x4A44",
+  "productId": "0x4447",
+  "keycodes": [],
+  "menus": [],
```
Comment (MaiTheSan) [inline src/thepanduuh/degenpad/degenpad.json:6]:
```suggestion
  "lighting":"none",
```

File: src/thepanduuh/degenpad/degenpad.json [hunk 4]
```diff
@@ -0,0 +1,112 @@
+{
+  "name": "Degenpad",
+  "vendorId": "0x4A44",
+  "productId": "0x4447",
+  "keycodes": [],
```
Comment (MaiTheSan) [inline src/thepanduuh/degenpad/degenpad.json:5]:
```suggestion

```

**Review by MaiTheSan — CHANGES_REQUESTED** [review]

**Comment by ThePanduuh** [comment]
Sorry about that. I must have done the V3 conversion in the V2 folder and was using that to test online. And then what was in the V3 folder was the beginning, prior to receipt of PCBs in my hands. 
Hopefully it's all good now.

**Comment by ThePanduuh** [comment]
Merged into QMK :)

**Comment by yiancar** [comment]
Cheers thanks!

**Pull request merged**